modules/core/keeper/msg_server_test.go

```diff
@@ -419,6 +445,14 @@ func (suite *KeeperTestSuite) TestHandleAcknowledgePacket() {
 				// replay should not error as it is treated as a no-op
 				_, err := keeper.Keeper.Acknowledgement(*suite.chainA.App.GetIBCKeeper(), suite.chainA.GetContext(), msg)
 				suite.Require().NoError(err)
+
```

Comment:
you could consider refactoring these test cases to use a function that asserts the expected test outcomes as in [this pr](https://github.com/cosmos/ibc-go/pull/5549/files) rather than checking against these `if` conditions, might be easier to read. but can also be done in a followup

Reply:
Yeah, we could do it. I think this better be a followup because these "if conditions" are already used commonly in this file for `async`. So, such a refactor would change unrelated testing code too